File: 4248-count-commas-in-range-ii/count-commas-in-range-ii.java
class Solution {
    public long countCommas(long n) {
       if(n<999) return 0;
      long count=0;

      long pro=1000;
     for(int i=0;i<5;i++){
        if(pro<=n){
            count+=(n-pro)+1;
        }
        else break;
        pro*=1000;
     }
    return count;
    }
}

File: 4248-count-commas-in-range-ii/count-commas-in-range-ii.cpp
class Solution {
public:
    long long countCommas(long long n) {
        long long ans=0;
        if(n<999)
            return 0;
        long long pow=1000;
        for(int i=0;i<5;i++){
            if(pow<=n){
                ans+=n-pow+1;
            }
            else 
                break;
            pow*=1000;
        }
        return ans;

    }
};